frontend/test/metabase/scenarios/question/settings.cy.spec.js

```diff
@@ -149,7 +149,7 @@ describe("scenarios > question > settings", () => {
       // This click doesn't do anything, but simply allows the array to be updated (test gives false positive without this step)
       cy.get("body").click("bottomRight");
 
-      findColumnAtIndex("Products → Category", 6);
+      findColumnAtIndex("Products → Category", 5);
```

Comment:
This should have always been 5, since we hide the "Total" column before it. It used to pass because previously, we still rendered draggable items, even if they weren't visible (set to display none)". Now, we don't render anything in the DOM for them.